### DISCUSSION TITLE
Can I use a roast profile to automate my HT 2K+?

### DISCUSSION THREAD

**Discussion by Mzedithj** [artisan-roaster-scope/artisan#872] [Q&A] [answered]
Hello, is it possible to load a previous roast profile Im happy with to automate a batch? So i don't have to change settings as i go?

**Comment by poundy** [comment]
Have you read the Artisan machine specific info at https://artisan-scope.org/machines/hottop/ ? That tells you what is possible.  And there's an older post at https://artisan-roasterscope.blogspot.com/2015/05/hottop-kn-8828b-2k.html that has more info as well.

Then you need to load the background roast.  Roast menu --> Background. Ctrl-B is also one of many ways of opening that dialog box.

**Comment by MAKOMO** [comment, marked as answer]
Maybe you are asking how to playback events (like burner changes) from a previous roast loaded as background profile: see the post on [Profile Templates](https://artisan-roasterscope.blogspot.com/2017/10/profile-templates.html)

**Reply by Mzedithj** [reply]
thank you for the response.
